RHEL/7/templates/static/bash/rsyslog_files_permissions.sh

```diff
@@ -0,0 +1,54 @@
+# platform = multi_platform_rhel
```

Comment:
And platform should be Red Hat Enterprise Linux 7.
Sorry that I didn't notice it in previous review.